<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstRunWizard</class>
 <widget class="QWizard" name="FirstRunWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="page1">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Your Game Install Path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gamePathEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>471</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseGameButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>140</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoDetectSteamButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>87</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Steam Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="gamePathStatusLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>335</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page2">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose storage location for Reliquary data:
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="workspacePathEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>701</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseWorkspaceButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>180</y>
      <width>87</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suggestGameDriveButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Game Drive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suggestHomeButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>87</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Home Directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page3">
   <widget class="QLabel" name="summaryLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>15</y>
      <width>1001</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
